client/src/components/Overview/AddToCart.jsx

```diff
@@ -17,15 +43,35 @@ const maxQtyArr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
 const AddToCart = () => {
   const state = useContext(StateContext);
+  const initialState = {
+    inventory: state.styles.filter((style) => style.style_id === state.selectedStyle)[0].skus,
+    size: 'Select Size',
+    quantity: 'Select Quantity',
+    sku: 'sku',
+  };
 
-  const filteredSizes = state.styles.filter((style) => style.style_id === state.selectedStyle)[0];
-  const [size, setSize] = useState('Select Size');
-  const inStock = Object.values(filteredSizes.skus).filter((sku) => sku.size === size)[0];
+  const [cartState, cartDispatch] = useReducer(cartReducer, initialState);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const options = {
+      url: '/api/cart',
+      method: 'post',
+      data: {
+        sku_id: Number(cartState.sku),
+      },
+    };
+    axios.request(options)
+      .then((response) => {
+        // eslint-disable-next-line no-console
+        console.log(response.data);
+      });
+  };
```

Comment:
This doesn't take into account when you add a higher quantity of items, but the post request definitely works.